digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -386,6 +388,29 @@ else if (defaultCommandTaskTTL.isPresent() && isRunningLongerThanTTL(previousSta
         return EcsCommandStatus.of(false, nextStatus);
     }
 
+    @VisibleForTesting
+    static Optional<String> getErrorMessageFromTask(String cluster, String taskArn, EcsClient client)
+    {
+        Optional<String> errorMessage;
+        try {
+            final Task task = client.getTask(cluster, taskArn);
+            if (task.getContainers().size() > 0) {
+                errorMessage = Optional.of(task.getContainers().get(0).getReason());
```

Comment:
ECS task always uses only 1 container?

Reply:
Good point!  IIUC, currently only 1 container is used in my understand.
But it might be better to support multiple containers for the future modification.
I will change it later.

Reply:
I fixed as getting reasons from multiple containers and concatenate them.